<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogUI</class>
 <widget class="QWidget" name="SettingsDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="language_gb">
     <property name="title">
      <string>Язык</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="ui_language_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Интерфейс</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="doc_language_label">
        <property name="text">
         <string>Документооборот</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="ui_language"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="doc_language"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="font_gb">
     <property name="title">
      <string>Шрифт</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="1">
         <widget class="QFontComboBox" name="font_family_value"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="font_size_label">
          <property name="text">
           <string>Размер</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="font_size_value">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="font_family_label">
          <property name="text">
           <string>Cемейство</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="example">
        <property name="text">
         <string>В чащах юга жил бы цитрус? Да, но фальшивый экземпляр!</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="show_text_main_menu">
     <property name="text">
      <string>Показывать названия кнопок главного меню</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
